_posts/2010-05-13-1.3.7-released.md

```diff
@@ -7,10 +7,10 @@ author_email: nick@gemcutter.org
 
 NOTE:
 
-http://rubygems.org is now the default source for downloading gems.
+https://rubygems.org is now the default source for downloading gems.
```

Comment:
These links probably shouldn't change, since this was `http` at the time of this post (I think?).